app/views/admin/litigations/_form.html.erb

```diff
@@ -41,5 +41,8 @@
     </div>
   </div>
 
-  <%= f.actions %>
+  <%= f.actions do %>
```

Comment:
so this is weird, arbre `f.actions` when in creating form in controller directly has different output than in custom forms.

Reply:
Yep, I was trying to find an upstream issue, no luck. Maybe I'll file a bug later on.